2021-06-10 10:20:44,641 - __main__ - ERROR - Error detected
Traceback (most recent call last):
  File "C:\Work\Работа\Питон\COVID_19_ 9880\Cоздание_исследований\create_public.py", line 31, in genlog
    self.create_public()
  File "C:\Work\Работа\Питон\COVID_19_ 9880\Cоздание_исследований\create_public.py", line 65, in create_public
    driver.find_element_by_css_selector("#itemForm\:tabView\:country_panel").click()
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 598, in find_element_by_css_selector
    return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=90.0.4430.212)

